### PR TITLE
Add warning about outdated Extending tutorial

### PR DESCRIPTION
**Pull request by DominikaK**

### CODE REVIEW

File: docs/tutorials/extending_platformui/extending_platformui_with_new_navigation.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Extending PlatformUI with new navigation
 
+!!! caution
+
+    For the time being this tutorial only covers v1 of eZ Platform.
```
Comment (mere-teresa) [inline docs/tutorials/extending_platformui/extending_platformui_with_new_navigation.md]: I'll be more direct and let "for the time being" go.

Comment (DominikaK) [reply docs/tutorials/extending_platformui/extending_platformui_with_new_navigation.md]: Hm, how about "currently"? I'm wondering if simply saying "This tutorial only covers" may sound like we're not planning to update it in future.

Comment (mere-teresa) [reply docs/tutorials/extending_platformui/extending_platformui_with_new_navigation.md]: We don't want to expose our intentions, we want to expose the facts. So I am comfortable with saying "this tutorial only covers".

Comment (DominikaK) [reply docs/tutorials/extending_platformui/extending_platformui_with_new_navigation.md]: Makes sense, but should I remove the next line then?

**Pull request merged**